framework-plugins/lisk-framework-chain-connector-plugin/test/unit/plugin.spec.ts

```diff
@@ -90,6 +91,14 @@ const getTestBlock = async () => {
 	});
 };
 
+const getApiClientMocks = () => {
```

Comment:
instead of creating function, you can just use 

`let apiClient;`
and in `beforeEach`, reassign the mocks